src/components/common/Footer.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import logo from "../../assets/images/logo2.png";
+import logo from "../../assets/images/Official-Logo-Web-Footer.png";

```

Comment:
Make sure you're consistent with naming conventions used on the app. In our case, we've named images like so `official-logo-web-footer.png`